### ISSUE TITLE
에러 해결 관련 이슈

### ISSUE THREAD

**Issue by woochanglee-KM** [bigwave-ai/test#1]
1. model.py에서 수정 부분
class Users(Base):
    __tablename__ = 'users' # 필수적으로 선언 -> table의 이름

    id = Column(String, primary_key=True)
    name = Column(String, nullable=False)
    email = Column(String, nullable=False)
    password = Column(String, nullable=False)
    phone = Column(String, nullable=True)

에서 name이 다른 곳에서는 username으로 쓰이므로, 다음과 같이 변경이 필요합니다.
    username = Column(String, unique=True, nullable=False)

또한 username은 unique=True를 걸어주는 편이 좋아보입니다.

2. 로그인 함수
def login(user, db):
  #logits
  #user가 있는지? 확인
  user =  auth_user(user.username, user.password, db)
  if not user:
    raise HTTPException(status_code = status.HTTP_401_UNAUTHORIZED, detail='user not found')
  data = {'username':user.id}
  token = create_access_token(data)

에서 data = {'username':user.id}의 user.id를 user.username으로 변경할 필요가 있습니다.

3. auth_user 함수
def auth_user(username:str, password:str, db):
  #받은 username으로 db에서 user테이블에서 id == username인지 조회
  #만약 있다면 'user있음'이라 알린다.
  #없다면, password도 맞는지 본다.
  #다만 password는 암호화 되어 저장되 있을 거기 때문에 그거 고려해서 풀어주고(?)
  #암호화된 password와 입력받는 password가 같은지 본다.
  #그리고 같다면 user 출력 한다.
  user = db.query(Users).filter(Users.id==username).first()
  if not user:
    return False
  if not verify_password(password, user.password):
    return False
  return user

에서  user = db.query(Users).filter(Users.id==username).first()부분의 User.id를 User.username으로 변경이 필요합니다.

4. get_user_info 함수
def get_user_info(id, db): #this id to token
    target = db.query(Users).filter(Users.id==id).first()
    return target

에서 id라고 되어있는 부분을 username으로 통일이 필요해보입니다.

**Comment by june3471** [comment]
1에서 name을 username으로 변경하는 것은 적절치 않습니다.
